tech_specs/component_definitions.md

```diff
@@ -25,24 +25,32 @@ In order to make these ends meet, we can identify the following levels:
 
 ### Control Center (eg. "Brewery Two")
 
-The top level grouping inside the GUI application. All sensors, actuators, and settings for a single service are somewhere in here. UI blocks are stored here, so multiple processes can share them.
+The top level grouping inside the GUI application. All sensors, actuators, and settings for a single service are somewhere in here. UI blocks are stored here, so multiple views can share them.
 
-### Dashboard (eg. "Fermentation" / "Overview" / "All temperature sensors")
+### View (eg. "Fermentation" / "Overview" / "All temperature sensors")
 
-A clearly defined subset of an activity. This involves a subset of all UI blocks, but generally uses all assigned UI blocks simultaneously.
+A canvas displaying one or more widgets. Views do not directly display data or actions: they contain all objects (widgets) that do.
 
-It should be possible that multiple dashboards share physical UI blocks.
+Views can contain other views. Views can share widgets.
 
-If you'd say an activity "has `N` processes", then you could create a separate dashboard for each process.
+### Widget (eg. UI Block / Graph)
+
+All discrete objects in the UI are widgets. UI blocks display state and actions, graphs display history. Widgets can be created and moved by users.
+
+The same widget can be present on multiple views.
 
 ### UI block (eg. PID, actuator, mutex)
 
-Performs a specific, abstract task. A UI block isn't busy brewing beer. It might keep temperature at a certain level, or even just toggle the heater.
+Represents a hardware component performing a specific, abstract task. A UI block isn't busy brewing beer. It might keep temperature at a certain level, or even just toggle the heater.
```

Comment:
As said before: the view block only represents a task. It doesn't perform any task, so it also doesn't keep temperature. It is a stateless and actionless display block. It can only display data or emit events.  If it represents a PID, the PID block in the firmware maintains the temp. They are linked in a separate step, so we should keep them separate. View block <--> active controller block.